### PR TITLE
Fix action bar in video form

### LINKED ISSUE #2950: User needs to click submit button two times to publish changes when editing video
As the title says. The same applies also to the `Cancel` button. Also, I couldn't reproduce this but Adam encounter a problem with validation: 
> After initial unsuccessful validation, the Title input component doesn't turn its styling back to the default one upon providing a valid input. It should.

This could be related. The goal of this task is to make sure that VideoForm working correctly and the validation works as the design requires. 
I think the problem more likely comes from how we managed the state there. We're setting the so-called `formStatus`, and I think this causes most of the issues. Maybe we should rethink how to manage the state in video form? 

Let's try:
- Remove `reference` prop from `Tooltip` and use `Tooltip` as a wrapper always
- Check whether we need `span` wrapper for `children` in `Tooltip`

### PR DESCRIPTION
**Pull request by rafalpawlow**
Fixes #2950

### CODE REVIEW

File: packages/atlas/src/components/ActionBar/ActionBar.styles.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const ActionBarContainer = styled.div<ActionBarContainerProps>`
   display: grid;
   gap: ${sizes(4)};
   padding: ${sizes(4)};
-  z-index: ${zIndex.transactionBar};
+  z-index: 9999999;
```
Comment (drillprop) [inline packages/atlas/src/components/ActionBar/ActionBar.styles.ts]: do we actually need to update `z-index` here?

**Review by drillprop — CHANGES_REQUESTED** [review]
It doesn't solve the issue entirely. I still need to click twice when I edit only the description.

**Review by drillprop — APPROVED** [review]
I went through all the pages and everything seems to work fine. The video form works as it should as well. LGTM!

**Review by kdembler — CHANGES_REQUESTED** [review]
It seems all tooltips are broken?

https://user-images.githubusercontent.com/12646744/178738762-9507219e-9b56-4f95-aa7a-f76dcbd011d7.mp4

I also don't see anything if I hover over the description field in video form

**Comment by rafalpawlow** [comment]
@kdembler I'm affraid we need that span inside tooltip component.

> I also don't see anything if I hover over the description field in video form

You mean title, right?

**Review by kdembler — APPROVED** [review]

**Pull request merged**